src/app/common/directives.js

```diff
@@ -1,40 +1,38 @@
-(function () {
+(function() {
```

Comment:
why are you removing the space before the `()` most linters want that space there